docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,123 @@
+sidebar_label: Networking & security
+---
+
+Aiven combines a multi-cloud strategy with a cloud-agnostic approach to make the [bring your own cloud (BYOC)](/docs/platform/concepts/byoc) experience not only versatile and cost-efficient but also secure.
+
+## Bastion proxy networking
+
+The [bastion](https://en.wikipedia.org/wiki/Bastion_host) proxy service acts as a trusted
+relay between the Aiven Management plane and your BYOC environment. A typical BYOC
+implementation results in workloads that are not directly accessible from external networks.
+The bastion proxy is located in a virtual private cloud (VPC) and facilitates the
+communication between Aiven Management and monitoring systems and your private workloads.
+The bastion proxy service is also responsible for facilitating a communication channel that
+allows the Aiven Management plane to launch services within your private cloud.
+
+### Traffic to the bastion
+
+Aiven Management plane traffic to your BYOC environment originates from redundant NAT
+gateways. Therefore, the bastion subnet needs to whitelist four static IPs on relevant
+ports.
+
+| Source                 | Destination    | Description                                                                                                                                   |
+| ---------------------- | ---------------| --------------------------------------------------------------------------------------------------------------------------------------------- |
+| Aiven Management plane | Bastion subnet | Aiven uses SSH from the Aiven Management plane to the bastion service node for troubleshooting, setup, or configuration tasks on the bastion. |
+| Aiven Management plane | Bastion subnet | Proxy service is used to tunnel the traffic from the Aiven Management plane to workload nodes.                                                |
+| Aiven Management plane | Bastion subnet | Aiven Management plane accesses the bastion service node to retrieve status information and perform routine operations.                       |
+
+### Traffic from the bastion
+
+| Source         | Destination      | Description                                                                                                                             |
+| -------------- | -----------------| --------------------------------------------------------------------------------------------------------------------------------------- |
+| Bastion subnet | Aiven Management | Communication channel with the Aiven Management plane used to collect metrics and logs from the bastion and workload nodes              |
+| Bastion subnet | Aiven Management | Required for the bastion and workload nodes to make calls to the Aiven Management plane                                                 |
+| Bastion subnet | AWS CloudFront   | Communication channel for downloading RPM packages for bastion and workload nodes setup                                                 |
```

Comment:
In the standard BYOC model, RPM downloads for workload nodes don't pass through the bastion, also I think "communication channel" suggests two-way communication or telemetry, but actually it's purely one-way downloading of RPMs that happens from Cloudfront, so:

```suggestion
| Bastion subnet | AWS CloudFront   | Used for downloading RPM packages for bastion nodes setup                                                 |
```